content/docs/practices/naming.md

```diff
@@ -75,13 +75,13 @@ The list is not exhaustive.
 | Family | Base unit | Remark | 
 | -------| --------- | ------ |
 | Time   | seconds   |        |
-| Temperature | celsius | Celsius is the most common one encountered in practice |
+| Temperature | celsius | _celsius_ is preferred over _kelvin_ for practical reasons. |
 | Length | meters | |
 | Bytes  | bytes | | 
-| Bits   | bytes | |
-| Percent | ratio(*) | Values are 0-1. <br/> *) Usually 'ratio' is not used as suffix, but rather A\_per\_B. Exceptions are e.g. disk\_usage\_ratio  |
+| Bits   | bytes | _Sic!_ |
+| Percent | ratio | Values are 0–1 (rather than 0–100). `ratio` is only used as a suffix for names like `disk_usage_ratio`. The usual suffix follows the pattern `A_per_B`. |
 | Voltage | volts | |
-| Electric current | amperes | | 
+| Electric current | amperes | |
 | Energy | joules | |
-| Weight | grams | |
+| Mass   | grams | _grams_ is preferred over _kilograms_ for practical reasons. |
```

Comment:
Ditto, it's due to the prefix.

Reply:
I'm not sure if it is worth it to discuss the detailed reason in a short note like this. I'll add something about the prefix to satisfy your remark.